lib/languages/cpp.js

```diff
@@ -36,11 +36,19 @@ CppParser.prototype.parse_function = function(line) {
     );
 
     var matches = xregexp.exec(line, regex);
+    var retval;
     if(matches === null) {
+      var reg = xregexp('(?P<name>' + this.settings.fnIdentifier + ')' + '\\s*\\(\\s*(?P<args>.*?)\\)');
```

Comment:
Could you explain why you choose to have an alternative regex for constructors? It seems like it would have been possible to alter the existing regex as well.